### DISCUSSION TITLE
Show tasks etc. in calendar view with start and end dates

### DISCUSSION THREAD

**Discussion by rmaen-aste** [kimai/kimai#4945] [Questions & Answers] [answered]
Hello! I'm evaluating Kimai for a use case where one of the core features is ability to plan projects and tasks in advance - in addition to time tracking.

Ideally, employees should be able to allocate time from their personal calendar to specific project/task/activity - possibly working on multiple different projects per day. A summary view for managers would be necessary as well. 

1) Is this something that could be achived with Kimai out of the box?  I looked at the tasks plugin, which could possibly work if the tasks were presented in a calendar, timeline or such a view. 

2) Alternatively, is it possible to extend the calendar view to allow users to just add custom "events" with start and end dates?

Any other suggestions to accomplish the task in hand are of course welcome. 

Thanks for your insights!

**Comment by kevinpapst** [comment, marked as answer]
Thanks for the idea 👍  
Right now this is not possible, but having a calendar for tasks would be a neat addition.
But that is nothing that is coming anytime near, unless someone wants to sponsor the development.

**Reply by niels-przybilla** [reply]
Sounds like a great idea. Do you have a rough estimation how much this development would be?

**Reply by kevinpapst** [reply]
Not in public. Also: the idea is not though through. What happens if you click a task ... ?
